### PR TITLE
Quickfix in `OnCreateUser`

### PR DESCRIPTION
**Pull request by newsiberian**

### CODE REVIEW

File: packages/reaction-accounts/server/methods/accounts.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ Accounts.onCreateUser(function (options, user) {
     // send a welcome email to new users,
     // but skip the first default admin user
     // (default admins already get a verification email)
-    if (!Meteor.users.find().count() === 0) {
+    if (!(Meteor.users.find().count() === 0)) {
```
Comment (jshimko) [inline packages/reaction-accounts/server/methods/accounts.js]: Why are the additional parens needed?  Also, a simpler equivalent to that could be:

```js
if (Meteor.users.findOne()) {
  Meteor.call("accounts/sendWelcomeEmail", shopId, user._id);
}
```

Comment (jshimko) [inline packages/reaction-accounts/server/methods/accounts.js]: Oh wait, nevermind.  Answered my own question there.  The simpler equivalent still stands though.

Comment (newsiberian) [inline packages/reaction-accounts/server/methods/accounts.js]: About using `findOne`: we need to block first and pass second, third, etc, so `findOne` can't help here.

**Comment by aaronjudd** [comment]
:+1: tested, merging.

**Pull request merged**